### PR TITLE
issue #24 : eslint rule: no comment

### PR DESCRIPTION
**Pull request by Mayuresh94**

### CODE REVIEW

**Comment by magwas** [comment]
The build fails for the same issue we had in taskmarket. Workaround is in its way.

**Comment by magwas** [comment]
The fix is in develop now. I cannot update your branch, please merge upstream develop here.

File: .eslintrc [hunk 1]
```diff
@@ -11,5 +11,13 @@
         "react": {
             "version": "detect"
         }
+    },
+    "plugins": [
+        "no-editor-code", 
+        "no-block-comments"
+    ],
+    "rules": {
+        "no-editor-code/no-region": "warn",
```
Comment (magwas) [inline .eslintrc:20]: use error, we wanna get strict

File: .eslintrc [hunk 2]
```diff
@@ -11,5 +11,13 @@
...
             "version": "detect"
         }
+    },
+    "plugins": [
+        "no-editor-code", 
+        "no-block-comments"
+    ],
+    "rules": {
+        "no-editor-code/no-region": "warn", 
+        "no-block-comments/no-block-comments": 2
```
Comment (magwas) [inline .eslintrc:21]: what does the `2` mean here?

Comment (Mayuresh94) [reply .eslintrc:21]: 0 = off, 1 = warn, 2 = error

**Comment by magwas** [comment]
Sorry, I have implemented this as example rule in eslint-plugin-kodekonveyor.

**Pull request closed without merging**